src/libraries/Microsoft.PowerFx.Core/Public/Values/ColorValue.cs

```diff
@@ -19,5 +20,10 @@ public override void Visit(IValueVisitor visitor)
         {
             visitor.Visit(this);
         }
+
+        public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettings settings)
+        {
+            throw new System.NotImplementedException();
```

Comment:
>throw new System.NotImplementedException(); [](http://example.com/codeflow?start=12&length=43)

We can followup in future PR, - this could be an RGB expression. 
(we should hit this with future tests in color.txt)

Reply:
Created issue #723 to track this pendency.